pytest test_scrapy_crawl.py fails if repo path contains "wiki"

If the local repo path for development and testing includes the word "wiki" some tests are failing:

E.g. if the repo path is "/home/moinwiki/moin" the command `pytest cli/_tests/test_scrapy_crawl.py` fails with
 
```
>           assert [] == get_open_wiki_files()
E           AssertionError: assert [] == [popenfile(pa...flags=557057)]
E             
E             Right contains one more item: popenfile(path='/home/moinwiki/moin/_test_artifacts/server-crawl.log', fd=10, position=746, mode='w', flags=557057)

```

The intention of this check is to warn if some files from wiki/data or wiki/index are still in use after the execution of cli commands.

--- a/src/moin/_tests/__init__.py
+++ b/src/moin/_tests/__init__.py
@@ -12,6 +12,7 @@ import socket
 from io import BytesIO
 from pathlib import Path
 import psutil
+import re
 
 from flask import g as flaskg
 
@@ -108,8 +109,11 @@ def get_dirs(subdir: str) -> tuple[Path, Path]:
 
 
 def get_open_wiki_files():
+    """check if any files in wiki/data or wiki/index are in use"""
     proc = psutil.Process()
-    files = [f for f in proc.open_files() if "wiki" in f.path]
-    for file in files:
-        print(f"open wiki {file}")
+    files = [f for f in proc.open_files() if re.search("/cli/wiki/", f.path)]
+    if files:
+        print(f"Process {proc.pid} {proc.name} has open files:")
+        for file in files:
+            print(f"    {file.path}")
     return files

--- a/src/moin/_tests/__init__.py
+++ b/src/moin/_tests/__init__.py
@@ -12,7 +12,6 @@ import socket
 from io import BytesIO
 from pathlib import Path
 import psutil
-import re
 
 from flask import g as flaskg
 
@@ -111,9 +110,9 @@ def get_dirs(subdir: str) -> tuple[Path, Path]:
 def get_open_wiki_files():
     """check if any files in wiki/data or wiki/index are in use"""
     proc = psutil.Process()
-    files = [f for f in proc.open_files() if re.search("/cli/wiki/", f.path)]
-    if files:
-        print(f"Process {proc.pid} {proc.name} has open files:")
-        for file in files:
-            print(f"    {file.path}")
+    files = []
+    for file in proc.open_files():
+        if Path(file.path).match("*/wiki/data/*") or Path(file.path).match("*/wiki/index/*"):
+            print(f"Wiki file open: {file.path}")
+            files.append(file)
     return files

